org.eclipse.xtend.lib/src/org/eclipse/xtend/lib/annotations/Accessors.xtend

```diff
@@ -89,6 +136,35 @@ class AccessorsProcessor implements TransformationParticipant<MutableMemberDecla
 
 	def dispatch void transform(MutableFieldDeclaration it, extension TransformationContext context) {
 		extension val util = new AccessorsProcessor.Util(context)
+		val annot = accessorsAnnotation
+		annot !== null && final
+			? {
```

Comment:
The coding style with `?` makes this hard to read.
I'd prefer to see simple if-blocks here.
Also the indentation of the warning messages look weird here. IMO they should be indented one level more than the addWarning(...) call.
```
   addWarning(
      '''
      text
      '''
   )
```